recipes-rubygems/rubygems-net-netconf_0.4.3.bb

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: MIT
+SUMMARY = "RubyGem: net-netconf"
+DESCRIPTION = "Updated and maintained fork of the Juniper Ruby NetConf client"
+HOMEPAGE = "https://github.com/kkirsche/net-netconf"
+
+LICENSE = "BSD 2"
```

Comment:
That should be `BSD-2-Clause` I'd say - btw worth an upstream patch - already had a couple of them - so actually this layer could also turn out to be helpful for upstream resources